src/helpers/formControl.ts

```diff
@@ -0,0 +1,10 @@
+import _ from 'lodash';
```

Comment:
Not sure this would work, the idea is to import only single function not the whole lodash library 🤔
```suggestion
import camelCase from 'lodash/camelcase';
```

Reply:
Fixed on: https://github.com/rosle/react-nimble-survey/pull/14/commits/f4b7cdfe65c28689a2f43b5c1fa5826db2f1708a ✨